src/pybind/mgr/dashboard/controllers/erasure_code_profile.py

```diff
@@ -42,13 +42,11 @@ class ErasureCodeProfileUi(ErasureCodeProfile):
     def info(self):
         '''Used for profile creation and editing'''
```

Comment:
I know it was added before but maybe you can fix it in this PR nevertheless?

```suggestion
        """
        Used for profile creation and editing
        """
```

Same issue a few lines above: https://github.com/ceph/ceph/blob/master/src/pybind/mgr/dashboard/controllers/erasure_code_profile.py#L14

Reply:
Done :)